ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java

```diff
@@ -121,6 +121,12 @@
      */
     EjbLogger DEPLOYMENT_LOGGER = Logger.getMessageLogger(EjbLogger.class, "org.jboss.as.ejb3.deployment");
 
+    /**
+     * A logger with the category {@code org.jboss.as.ejb3.iiop} used for deployment
+     * log messages
+     */
+    EjbLogger IIOP_DEPLOYMENT_LOGGER = Logger.getMessageLogger(EjbLogger.class, "org.jboss.as.ejb3.iiop");
```

Comment:
This should just use DEPLOYMENT_LOGGER.

Reply:
Thanks, Brian. I have changed accordingly.